tests/sort_customers_by_name_test.py

```diff
@@ -0,0 +1,32 @@
+import allure
+
+from pages.list_customers_page import ListCustomersPage
+from pages.manager_page import ManagerPage
+
+
+@allure.feature('globalsqa.com')
+@allure.title('Сортировка клиентов по имени (First Name)')
+@allure.description("""
+    Цель: Проверить сортировка клиентов по имени
+
+    Предусловие: Открыть браузер
+
+    Шаги:
+    1. Открыть страницу "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"    
+    2. Дважды нажать на заголовок таблицы "First Name" для сортировки пользователей по имени
+    3. Проверить, что имена отсортирована в алфавитном порядке
+    """)
+def test_sort_by_first_name(driver):
+    with allure.step("Открытие страницу 'https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager'"):
+        list_customers_page = ListCustomersPage(driver)
+        list_customers_page.open()
+
+    with allure.step("Сортировка пользователей по имени"):
+        manager_page = ManagerPage(driver)
+        manager_page.click_on_item_menu("Customers")
```

Comment:
`class ListCustomersPage(ManagerPage):`
```suggestion
    with allure.step("Сортировка пользователей по имени"):
        list_customers_page.click_on_item_menu("Customers")
```